### PR TITLE
Added Authentication and Authorisation

### PR DESCRIPTION
**Pull request by Yashwanth1129**
Added Authentication and Authorization

### CODE REVIEW

File: PensionManagementUserLoginService/Models/Context/AuthDbContext.cs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    public class AuthDbContext: IdentityDbContext
+    {
+        public AuthDbContext(DbContextOptions<AuthDbContext> options) : base(options)
+        {
+
+        }
+        protected override void OnModelCreating(ModelBuilder builder)
+        {
+            base.OnModelCreating(builder);
+            var readerRoleId = "92327fd8-20fc-42d8-9bd8-536a746bc3ec";
```
Comment (Anirudh-2024) [inline PensionManagementUserLoginService/Models/Context/AuthDbContext.cs]: is this the default id or you have provide it

File: PensionManagementUserLoginService/Models/Context/AuthDbContext.cs [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            var writerRoleId = "ab185359-929b-48ef-85b3-6acb0637dcfd";
```
Comment (Anirudh-2024) [inline PensionManagementUserLoginService/Models/Context/AuthDbContext.cs]: same here is this the default if or you have provided it

File: PensionManagementUserLoginService/Models/Context/AuthDbContext.cs [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+                    Name="Writer",
+                    NormalizedName="Writer".ToUpper(),
+                    ConcurrencyStamp=writerRoleId
+                },
+            };
+
+            builder.Entity<IdentityRole>().HasData(roles);
+
+
+            var adminUserId = "7ace62fb-9191-4678-946c-3de3f42347da";
```
Comment (Anirudh-2024) [inline PensionManagementUserLoginService/Models/Context/AuthDbContext.cs]: even this ?

File: PensionManagementUserLoginService/Models/Context/AuthDbContext.cs (hunk 1 above)
Comment (Yashwanth1129) [reply PensionManagementUserLoginService/Models/Context/AuthDbContext.cs]: we have generated new Guid Id using visual studio in C# interactive
where it is a unique identifier for the readerroleId

File: PensionManagementUserLoginService/Models/Context/AuthDbContext.cs (hunk 2 above)
Comment (Yashwanth1129) [reply PensionManagementUserLoginService/Models/Context/AuthDbContext.cs]: we have generated new Guid Id using visual studio in C# interactive
where it is a unique identifier for the writerroleId

File: PensionManagementUserLoginService/Models/Context/AuthDbContext.cs (hunk 3 above)
Comment (Yashwanth1129) [reply PensionManagementUserLoginService/Models/Context/AuthDbContext.cs]: Generated new Guid Id using visual studio in C# interactive
where it is a unique identifier for the admin

File: PensionManagementUserLoginService/Models/Context/AuthDbContext.cs (hunk 1 above)
Comment (Anirudh-2024) [reply PensionManagementUserLoginService/Models/Context/AuthDbContext.cs]: Guid.New().ToString('N');

**Review by Anirudh-2024 — APPROVED** [review]
Approved, LGTM

**Pull request merged**